sqlitecluster/SQLiteNode.cpp

```diff
@@ -328,9 +328,9 @@ bool SQLiteNode::shutdownComplete() {
     // Next, see if we're timing out the graceful shutdown and killing non-gracefully
     if (_gracefulShutdownTimeout.ringing()) {
         SWARN("Graceful shutdown timed out, killing non gracefully.");
+        auto lock = _escalatedCommandMap.scopedLock();
```

Comment:
this scope now includes _changeState, is that intentional or should we scope this to only be the if block?

Reply:
It doesn't really matter too much. we could add a scope block, but _changeState shouldn't take very long and happens infrequently anyway.